toolkit/tools/pkg/imagecustomizerlib/testdata/scripts/consume-space.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
```

Comment:
Please add MS Copyright header

Reply:
It is probably okay to leave off the copyright header over test/sample configs.